nerfstudio/data/datasets/base_dataset.py

```diff
@@ -46,7 +46,7 @@ class InputDataset(Dataset):
     exclude_batch_keys_from_device: List[str] = ["image", "mask"]
     cameras: Cameras
 
-    def __init__(self, dataparser_outputs: DataparserOutputs, scale_factor: float = 1.0, cache_images: bool = False):
+    def __init__(self, dataparser_outputs: DataparserOutputs, scale_factor: float = 1.0):
```

Comment:
oh doh did this not end up being helpful?  I guess the I/O can get hidden enough otherwise?

Reply:
So originally there is no increase in training speed when loading the image from disk vs. caching the compressed image to RAM; I was getting about 25 minutes for nerfacto to complete 30,000 training steps for both methods. However, the caching of compressed images to RAM took an extra ~2 minutes in the setup time

![image](https://github.com/user-attachments/assets/99514e25-ede8-44ed-ac56-756561c23212)

I actually think I may have made a mistake in deleting this because I do have an M.2 SSD drive which is very very speedy. I think this could actually make a difference if a user has a much slower drive they are reading from, so I will lean towards adding this back